assets/prefabs/buildings/residentialHouse.prefab

```diff
@@ -1,16 +1,19 @@
 {
-    "GenericBuilding" : {
-     "name" : "House",
-     "generatorNames" : ["RectHouseGenerator"],
-     "zone" : "RESIDENTIAL",
-     "minSize" : [14, 14],
-     "maxSize" : [22, 22],
-        "isEntity" : true
-    },
-    "PotentialHome": {
-        "maxCitizens": 1
-    },
-    "SimpleSource": {
-        "needType": "REST"
-    }
+  "GenericBuilding": {
+    "name": "House",
+    "templateNames": [
+      "MetalRenegades:oldwestResidentialHouse1",
+      "MetalRenegades:oldwestResidentialHouse2"
```

Comment:
This will place both buildings on top of each other, one after the other. See https://github.com/Terasology/DynamicCities/pull/104 where I've added some docstrings to clarify this behavior.